akka-docs/src/main/paradox/scala/stream/stages-overview.md

```diff
@@ -21,6 +21,17 @@ If the iterator perform blocking operations, make sure to run it on a separate d
 
 ---------------------------------------------------------------
 
+@@@ div { .group-scala }
```

Comment:
This seems the wrong way around: 'apply' is the Scala version, 'from' is the Java one.